gems/sorbet/test/snapshot/BUILD

```diff
@@ -65,9 +65,6 @@ snapshot_tests(
 
 test_suite(
     name = "snapshot",
-
-    # NOTE: this is manual to avoid being caught with `//...`
-    tags = ["manual"],
```

Comment:
Removing `manual` here seems unusual: https://docs.bazel.build/versions/main/test-encyclopedia.html#tag-conventions

Some of the other tag removals ("external") are also defined at the above link; why are we removing all the tags?

Reply:
I think the difference is that these tags aren't applied to the `test_suite` target, but are used to find tests to include in it. This is pretty confusing given that in every other use `tags` applies the tags to the target: https://docs.bazel.build/versions/main/be/general.html#:~:text=Only%20test%20rules,by%20visibility%20constraints).

Reply:
Though now that I'm reading beyond what I linked, I'm seeing this:

> The manual tag keyword is treated differently than the above by the "test_suite expansion" performed by the bazel test command on invocations involving wildcard target patterns. There, test_suite targets tagged "manual" are filtered out (and thus not expanded). This behavior is consistent with how bazel build and bazel test handle wildcard target patterns in general.

Which indicates that the `"manual"` tag is treated not as a filter for finding tests, but instead is used for having the `test_suite` skipped during wildcard expansion. That's pretty unfortunate, but does explain why all of a sudden I started seeing compiler targets showing up in this build :)

I'll put the manual uses back in, but remove the other cases where the tags might accidentally widen the scope of the test suite. Thanks for pointing this out!